collect_app/src/main/res/drawable/ic_instance_marker_finalized.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Are these icons coming from Studio's Vector Assests tool? If so could we leave the names the way they are when they are generated? That way it's easier to work out whether we already have an icon in our resources or not.

Reply:
Going through the Vector Assets tool gives name `ic_room_black_24dp` no matter the tint. The icon already exists without a tint as `ic_place_black` and is used for warning dialogs related to location.

For this feature, there are 4 colored variants needed as you can see in [`getDrawableIdForStatus`](https://github.com/opendatakit/collect/pull/3447/files#diff-8419b5f9069e6719d3038c7dca542854R211). Is there a way to do that other than introducing 4 resources?

Reply:
I'll add that the marker gets set by the `setMarkerIcon` method defined by the `MapFragment` interface. I believe that to use a single black icon and tint it, we'd have to introduce a new method there and make sure hat all three mapping engines (Mapbox, OSM, Google Maps) all support tinting across all API levels. Does that seem worth it, @seadowg?

Reply:
Ah I see because it takes a resource `int` I see. Yeah right now that does feel like a bigger disruption. For the moment can we just have them named like `ic_room_red_24dp`, `ic_room_blue_24dp` etc?

Just a not on tinting though: My impression is that we'd be able to use [`DrawableCompat.wrap`](https://developer.android.com/reference/androidx/core/graphics/drawable/DrawableCompat.html?hl=en#wrap(android.graphics.drawable.Drawable)) to create "tintable" drawables back to at least API 14. I guess that it might not work with map engines for some reason but I think we could probably just test at our min API level.